web/modules/custom/ai_screening_project_track/src/Form/ThresholdsForm.php

```diff
@@ -67,18 +68,18 @@ public function buildForm(array $form, FormStateInterface $form_state): array {
           '#attributes' => ['class' => ['grid', 'gap-4', 'grid-cols-2', 'border-bottom']],
         ];
 
-        $form['project_track'][$termId]['term_wrapper'][$key]['dimensions_wrapper']['approved-' . $termId . '-' . $key] = [
+        $form['project_track'][$termId]['term_wrapper'][$key]['dimensions_wrapper'][Evaluation::APPROVED->getAsLowerCase() . '-' . $termId . '-' . $key] = [
```

Comment:
Consider adding a `buildThresholdKey`, say, function in `ProjectTrackTypeHelper` to build this key along with a function to split/parse a key, e.g.

```php
private const string THRESHOLD_KEY_SEPARATOR = '-';

public static function buildThresholdKey(Evaluation $evaluation, int $termId, int $dimension): string {
  return implode(self::THRESHOLD_KEY_SEPARATOR, [$evaluation->value, (string) $termId, (string) $dimension);
}

public static function getThresholdKeys(string $key): array {
  $values = split(self::THRESHOLD_KEY_SEPARATOR, $key);
  assert(count($value) !== 3);
  return [
    Evaluation::from($values[0]),
    (int) $values[1],
    (int) $values[2],
  ];
}
```